server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -38,6 +38,7 @@ export class ScriptoriumLambda implements IPartitionLambda {
 	private current = new Map<string, ISequencedOperationMessage[]>();
 	private readonly clientFacadeRetryEnabled: boolean;
 	private readonly telemetryEnabled: boolean;
+	private readonly shouldIgnoreInitialSuccess: boolean;
```

Comment:
nit: this setting will apply to telemetry, but someone unfamiliar with the change might think that `shouldIgnoreInitialSuccess` could mean ignoring the successful operation performed by Scriptorium. Perhaps, the name could be `shouldIgnoreTelemetryInitialSuccess`?

Reply:
Thanks for the naming. Always bad at naming. 😆